### PR TITLE
join SNP INDEL

### PR DESCRIPTION
**Pull request by ElisabetThomsen**
INDEL were not in the output vcf
Change: Join SNP and INDEL after VQSR so you get both in the output vcf

### CODE REVIEW

**Comment by olavurmortensen** [comment]
Vit kunnu líka so gott diskutera hesar broytingarnar her.

Tað er rætt sum tú sigur í mailinum, tú skalt brúka `set sample, ...` so man heldur skil á samples.

Læt meg frætta tá tú hevur testað at tað riggar.

File: main.nf [hunk 1]
```diff
@@ -495,12 +495,50 @@ process annotate_rsid {
...
+    -V $vcf \
+    -select-type INDEL \
+    -O "indel.vcf"
+    --java-options "-Xmx${task.memory.toGiga()}g -Xms${task.memory.toGiga()}g"
+    """
+}
+
+// Hard filter variants, adding various filter tags to the "FILTER" field of the VCF.
+// FIXME: I'm getting warnings that MQRankSum and ReadPosRankSum don't exist.
+process hard_filter_SNPs {
```
Comment (olavurmortensen) [inline main.nf]: @ElisabetThomsen Tað er betur vissi `hard_filter_SNPs` og `HardFilterINDELs` navngevingin líkist.

**Comment by olavurmortensen** [comment]
In retrospect, it's very unprofessional to write in anything other than english on GitHub :sweat_smile:

File: main.nf (hunk 1 above)
Comment (ElisabetThomsen) [reply main.nf]: I have fixed this 👍

**Comment by olavurmortensen** [comment]
SNP and indel merging added to `main.nf` and `joint_genotyping.nf`. `main.nf` is tested, but `joint_genotyping.nf` is not.

**Pull request merged**